### PR TITLE
doc/website/blog: terraform v0.5.0 announcement

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+the desired schema. The scheme of this URL determines which schema loader will be used, for instance:
+
+* `file://schema.sql` - loads the schema from a local SQL file.
+* `mysql://root:pass@localhost:3306/myapp` - loads the schema from an existing MySQL database.
+* `ent://service/ent/schema` - loads the schema from the schema of an [Ent](https://entgo.io/)
+  project.
+
+### Managing database schemas in plain SQL using Terraform
+
+In the following example, we will show to use Terraform and the Atlas provider to manage
```
Comment (rotemtam) [inline doc/website/blog/2023-04-20-terraform-v047.md:71]:
```suggestion
In the following example, we will show how you can use Terraform and the Atlas provider to manage
```

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+a MySQL database schema in plain SQL.
+
+Let's start by creating a Terraform configuration file named `main.tf` installing two
```
Comment (rotemtam) [inline doc/website/blog/2023-04-20-terraform-v047.md:74]:
```suggestion
Let's start by creating a Terraform file named `main.tf` installing the Atlas
```

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+Terraform providers:
```
Comment (rotemtam) [inline doc/website/blog/2023-04-20-terraform-v047.md:75]:
```suggestion
Terraform provider:
```

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+---
+:::info TL;DR
+
+You can now use plain SQL to define the desired state of your database schema.
+  
+[See an example](#managing-database-schemas-in-plain-sql-using-terraform)
+
+:::
+
+Earlier this week, we released [v0.4.7](https://github.com/ariga/terraform-provider-atlas/releases/tag/v0.4.7)
```
Comment (a8m) [inline doc/website/blog/2023-04-20-terraform-v047.md:14]: Sounds like a big feature. Why not release a new minor version? v0.5.0

**Review by a8m — APPROVED** [review]

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+}
+```
+
+### Defining the desired state of the database schema in plain SQL
+
+In earlier versions of the Atlas Provider, the `atlas_schema` resource required the user
+to provide an [Atlas HCL](/atlas-schema/sql-resources) file which describes the desired
+state of the database schema. Many users found this syntax, which resembles Terraform's
+own, to be clean and concise. However, others disliked it and asked for a way to define
+the desired state in plain SQL.
```
Comment (masseelch) [inline doc/website/blog/2023-04-20-terraform-v047.md:55]: Dislike but as well limited support on HCL. With SQL, we now support everything SQL does support out if the box, which atlas does not yet.

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+
+To support this use-case, and many others, we have announced 
+[support for "schema loaders"](http://localhost:3000/blog/2023/03/13/atlas-v010#schema-loaders) -
```
Comment (masseelch) [inline doc/website/blog/2023-04-20-terraform-v047.md:58]: Remove localhost

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+# ...
+
+Terraform has been successfully initialized!
+```
+Finally, we can run `terraform apply` to create the database schema:
+
+```console
+terraform apply
+```
+Terrform will print the following plan:
```
Comment (masseelch) [inline doc/website/blog/2023-04-20-terraform-v047.md:153]:
```suggestion
Terraform will print the following plan:
```

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 8]
```diff
@@ -0,0 +1,367 @@
...
+atlas_schema.myapp: Modifying... [id=mysql:///myapp]
+atlas_schema.myapp: Modifications complete after 4s [id=mysql:///myapp]
+
+Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
+```
+
+## Conclusion
+
+In this tutorial, we have seen how to use Terraform to manage the schema of a MySQL database
+using the Atlas provider with plain SQL.  Using this workflow, teams can bridge the gap between
```
Comment (masseelch) [inline doc/website/blog/2023-04-20-terraform-v047.md:361]:
```suggestion
using the Atlas provider with plain SQL. Using this workflow, teams can bridge the gap between
```

**Review by masseelch — COMMENTED** [review]
👍

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+driver to spin up a temporary database container:
+
+```hcl
+provider "atlas" {
+  dev_url = "docker://mysql/8/myapp"
+}
+
+resource "atlas_schema" "mysql" {
+  url = "mysql://root:pass@localhost:3306/myapp"
+  schema = data.atlas_schema.sql.schema
```
Comment (giautm) [inline doc/website/blog/2023-04-20-terraform-v047.md:126]:
```suggestion
  hcl = data.atlas_schema.sql.hcl
```

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 10]
```diff
@@ -0,0 +1,367 @@
...
+  depends_on = [docker_container.mysql]
```
Comment (giautm) [inline doc/website/blog/2023-04-20-terraform-v047.md:127]: You don't have any resource for docker container in the example

File: doc/website/blog/2023-04-20-terraform-v047.md [hunk 11]
```diff
@@ -0,0 +1,367 @@
...
+}
+```
+
+Now that we have defined our Terraform configuration, we can run `terraform init` to
```
Comment (giautm) [inline doc/website/blog/2023-04-20-terraform-v047.md:131]: Please add the full content of `main.tf` here to easy copy & paste

**Pull request merged**